daemon/albatross_console.ml

```diff
@@ -251,15 +251,16 @@ let handle s addr =
     | Ok (header, `Command (`Console_cmd cmd)) ->
       begin
         let name = header.Vmm_commands.name in
-        match Vmm_core.Name.path name, Vmm_core.Name.name name with
-        | _, None ->
-          (Vmm_lwt.write_wire s (header, `Failure "path used instead of name") >>= function
-            | Ok () -> Lwt.return_unit
-            | Error _ -> Vmm_lwt.safe_close s)
-        | path, Some name ->
-          match cmd with
-          | `Console_add n ->
-            begin
+        let path = Vmm_core.Name.path name in
+        let lbl = Vmm_core.Name.name name in
+        match cmd  with
+        | `Console_add n ->
```

Comment:
We could match on `cmd, lbl` and then handle `lbl = None` for `` `Console_add _ | `Console_subscribe `` to reduce duplicated code a little. Then we would have a match `` `Console_list_inactive, _ ``